### ISSUE TITLE
`url` item is being deleted

### ISSUE THREAD

**Issue by vchrombie** [Groninger-Bugbusters/grimoirelab-perceval-coveralls#3]
I see that the `url` item is empty, so it was being removed.

https://github.com/Groninger-Bugbusters/grimoirelab-perceval-coveralls/blob/ce191ed0468e030899fb1e4b991d49a4e34e3c2b/perceval/backends/coverage/coveralls.py#L69-L71

Is it empty always in github, does it work fine with gitlab or bitbucket? I'm thinking the URL being empty might be some issue with the coveralls service (maybe it is a bug which they are working on).

But, as per my understanding, the goal of perceval is to do one thing which is fetching data from software repositories. I think it is good we avoid tampering with the data returned by the API. The general idea is that the pre-processing of the data (retrieved by perceval) is done by [GrimoireELK](https://github.com/chaoss/grimoirelab-elk) as per the [GrimoireLab architecture](https://github.com/chaoss/grimoirelab-tutorial/blob/tesseract/docs/components/index.md). So, I think it is better we don't delete any item and leave it the way it is returned, at least with Perceval.

I would very much want to hear your views on this.

**Comment by DavidVisscher** [comment]
Thanks for the feedback! I agree with your assessment. :+1: 

At one point we were considering allowing archiving for this backend. 
At that point, these three lines were added to work around an Exception we were getting during pickling of the items.

Since we decided not to support that, these lines can be removed.
I have done so in the following commit: https://github.com/Groninger-Bugbusters/grimoirelab-perceval-coveralls/commit/bd00cf235e6aeea0de82c2906d04c8a6d4674800
